.jenkins/pytorch/test.sh

```diff
@@ -173,10 +173,10 @@ test_aten() {
       SUDO=sudo
     fi
 
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libc10* build/bin
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libcaffe2* build/bin
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libmkldnn* build/bin
-    ${SUDO} ln -s "$TORCH_LIB_PATH"/libnccl* build/bin
+    ${SUDO} ln -s "$TORCH_LIB_PATH"/libc10* build/bin || true
```

Comment:
I actually dont know what is the expected behavior when libc10* already exist. but here is the command to check if they do
```
if compgen -G "build/bin/libc10*" > /dev/null; then
  # Do something if they exist
else
  ${SUDO} ln -s "$TORCH_LIB_PATH"/libc10* build/bin
fi
```

Reply:
I was just going off [your comment](https://github.com/pytorch/pytorch/pull/57622#issuecomment-840612300):

> The way GHA downloads build artifact and unzip them back out the test phase makes it unnecessary to symlink those site-package binary back into the repo pytorch/build/bin folder.

but I don't know what's going on here so let me know if I should change this